src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -78,6 +73,32 @@ export const ResearchProfilePanel = ({ person, isLoadingPerson }: ResearchProfil
                 </Fragment>
               ))}
             </Box>
+            {personKeywords.length > 0 && (
+              <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+                <Typography variant="h3">{t('my_page.my_profile.field_and_background.field')}</Typography>
+                <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '0.25rem' }}>
+                  {personKeywords.map((keyword) => (
+                    <Chip
+                      sx={{
+                        width: 'fit-content',
```

Comment:
Kan bli litt funky med veldig lange nøkkelord, men tenker det er bedre enn å sette `max-width` da det øker bredden på hele panelet.

Reply:
Syns det ser greit nok ut :)
Stusser derimot over at disse skal se ut likt som knappene, for meg tyder det på at disse er klikkbare 🤔 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/4d425483-fbc8-4832-970e-9010d54aa305)

Reply:
Godt poeng :thinking: Skal ta en prat med Cecilie

Reply:
Bakgrunnsfargen ble fjernet for å vise at de er forskjellige. Tenker at de fortsatt er prinsipielt ganske lik, men det er et steg i riktig retning.

Reply:
syns det ble bedre nå ja 👍